src/main/kotlin/app/widgets/welcomeBack/WelcomeBack.kt

```diff
@@ -0,0 +1,56 @@
+package app.widgets.welcomeBack
+
+import app.MyApp.Companion.Colors
+import app.ui.styles.ButtonStyles
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.geometry.Pos
+import javafx.scene.layout.HBox
+import javafx.scene.paint.Color
+import javafx.scene.text.FontWeight
+import tornadofx.*
+import app.widgets.profileIcon.ProfileIcon
+import tornadofx.FX.Companion.messages
+import java.awt.Button
+import java.util.*
+
+class WelcomeBack : HBox() {
+    private val rad = 125.0
+    private val bigIcons = borderpane() {
+
+        alignment = Pos.CENTER
+
+        style {
+            backgroundColor += c(Colors["base"])
+        }
+        val iconHash = ProfileIcon("12345678901", 150.0, true)
```

Comment:
Should profileIcon be a parameter to this widget? (or Node?) especially since we don't want to hardcode values in widgets